image-viewer.py

```diff
@@ -1,37 +1,40 @@
-from tkinter import Tk, Label, Button
+from tkinter import Tk, Label, Button, Menu, filedialog
 from PIL import Image, ImageTk
 from os import listdir
 from os.path import isfile
 
 class ImageViewer():
     def __init__(self):
+                     
         self.current_image_index = 0
         self.image_files = []
+        self.blank_img = Image.new('RGBA',(500,300),(0,0,0,0)) # Blank Image to display if no other images are found   
```

Comment:
I'd say a lot of the comments that are in here aren't actually needed since the code is "self-documented", meaning what is done can easily be understood from the variable name or what the code is doing, even for beginners, so I'd suggest removing them :).